visualisation/draw.py

```diff
@@ -128,17 +128,24 @@ def choosing_colorway():
         colorscale_names, index=31)
 
     chosen_color = getattr(modules_colors_d[chosen_module_color], f'{all_colors[plots_color]}')
-
     return chosen_color
 
 
 @st.cache
 def get_colors_names(chosen_module_color):
     colorscale_names = []
-
+    
     # Getting all colors from modules
     colorscale_names.extend([f'{name}' for name, body
                              in inspect.getmembers(getattr(px.colors, chosen_module_color))
                              if isinstance(body, list)])
-
+    
     return colorscale_names
+
+
+def adjust_plot_colors_n_templates():
+    with st.beta_expander("Customize your chart"):
+        plots_color = choosing_colorway()
+        template = choose_template()
+    
+    return plots_color, template
```

Comment:
to też bym dał w `vis.py` bezpośrednio.

`choosing_cośtam` to genialna nazwa. Zastosujmy tę samą konwencję funkcjach gdzie wybierasz window i degree do wygładzania <3